### ISSUE TITLE
Portfolio-Mobile-Menu Issue

### ISSUE THREAD

**Issue by afiamoah** [Aliahmad-karimi/My-Portfolio#14]
Fantastic job done
# Suggestion
the `const clos = document.querySelector('.clos-button');` inside the script.js file can be renamed to "CloseButton" for clear description.
Good job done

**Comment by OmarMWarraich** [comment]
In the languages section above contact section, kindly align the languages div.

**Comment by kal2944** [comment]
I agree with those suggestions otherwise everything is good for me.  You did a great job!

**Comment by Aliahmad-karimi** [comment]
Thanks for your advice guys I will try to implement the changes u want
